services/app-api/src/resolvers/healthPlanPackage/submitHealthPlanPackage.ts

```diff
@@ -225,21 +230,36 @@ export function submitHealthPlanPackageResolver(
         ) {
             const errMessage = `Attempted to submit an already submitted package.`
             logError('submitHealthPlanPackage', errMessage)
-            throw new UserInputError(errMessage) // TODO: This is should be a custom ApolloError such as INVALID_PACKAGE_STATUS or ACTION_UNAVAILABLE, not user input error since doesn't involve form fields the user controls
+            throw new GraphQLError(errMessage, {
+                extensions: {
+                    code: 'INVALID_PACKAGE_STATUS',
```

Comment:
After our informal eng meeting I think we said this is supposed to be `INTERNAL_SERVER_ERROR` for code as well right?  INVALID_PACKAGE_STATUS is the cause not the code is where we landed. 

And USER_INPUT_ERROR (what was originally here in the diff) is a code but the wrong one for this case.